packages/@react-spectrum/card/src/CardView.tsx

```diff
@@ -195,6 +196,20 @@ function InternalCard(props) {
     focusMode: 'cell'
   }, state, unwrappedRef);
 
+  // Prevent space key from scrolling the CardView if triggered on a disabled item or on a Card in a selectionMode="none" CardView.
+  let allowsInteraction = state.selectionManager.selectionMode !== 'none';
+  let isDisabled = !allowsInteraction || state.disabledKeys.has(item.key);
```

Comment:
On second thought, might want to remove all this card specific stuff from my other PR so the scope for this PR is more defined